transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -470,18 +523,26 @@ static const JNINativeMethod fixed_method_table[] = {
 static const jint fixed_method_table_size = sizeof(fixed_method_table) / sizeof(fixed_method_table[0]);
 
 static jint dynamicMethodsTableSize() {
-    return fixed_method_table_size + 1; // 1 is for the dynamic method signatures.
+    return fixed_method_table_size + 2; // 1 is for the dynamic method signatures.
```

Comment:
=>     `return fixed_method_table_size + 2; // 2 is for the dynamic method signatures.`?

Reply:
yes exactly. it was 1 because we only had one before

Reply:
I meant changing the 1 in the comment to 2 as well, no?

Reply:
ah yeah... did miss this 👎

Reply:
😄